### PR TITLE
updating the node bootstrapping guide

### PR DESCRIPTION
**Pull request by vishalchangrani**
Some changes that I would like to do after reviewing https://github.com/onflow/flow/pull/862/.

## Description

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Updated relevant documentation 
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: docs/content/node-operation/node-bootstrap.mdx [hunk 1]
```diff
@@ -262,7 +279,7 @@ This is the recommended way to start your node for the first time.
 3. Configure Dynamic Startup by adding flags:
 ```shell:title=ExampleDynamicStartupFlags
   ... \
-  --dynamic-startup-access-address=access.flow.com:9001 \
```
Comment (vishalchangrani) [inline docs/content/node-operation/node-bootstrap.mdx:265]: `access.flow.com` doesn't exist today. So, I replaced it with access-001. Plus, the key is specific to that node.

File: docs/content/node-operation/node-bootstrap.mdx [hunk 2]
```diff
@@ -49,12 +49,11 @@ a806ebf3f1b4d866bead61f8e8049f534c54006d0b903f0d5ea7045aa4bcd791  ./boot-tools/t
...
 ### Generate Your Node Keys
 
-#### Addresses
+#### Network Address
 
 <Callout type="info">
 
-You will need to have a real address, for example, a CNAME pointed at the
-node or just the IP address e.g. `example.com:3569` or `189.23.42.12:3569`.
+Use a fully qualified domain name for the network address. Please also include the port number in the network address e.g. `example.com:3569`
```
Comment (jordanschalm) [inline docs/content/node-operation/node-bootstrap.mdx:56]: 👍

File: docs/content/node-operation/node-bootstrap.mdx (hunk 1 above)
Comment (jordanschalm) [reply docs/content/node-operation/node-bootstrap.mdx:265]: The public key I have here is not a real value either, it's just a placeholder/example. We would need to change that as well to make this a usable example. 

If we use the `mainnet16` address, we will need to update it every spork. I remember we talked about adding a `secure.access...` address for use in the DKG, which would be:
* the same address sporks
* point to the same node across sporks (eg. AN1)

Do you remember if we ever got that set up?

File: docs/content/node-operation/node-bootstrap.mdx [hunk 3]
```diff
@@ -274,7 +291,7 @@ Once the node has bootstrapped, these flags may be removed.
 ### Manually Provisioned Root Snapshot
 
 You can also provision the root snapshot file manually, then start the node without configuring Dynamic Startup.
-A snapshot can be obtained from any Access Node using Flow CLI.
+A snapshot can be obtained from any Access Node using [Flow CLI](https://docs.onflow.org/flow-cli/).
```
Comment (jordanschalm) [inline docs/content/node-operation/node-bootstrap.mdx:294]: 🙇

File: docs/content/node-operation/node-bootstrap.mdx (hunk 1 above)
Comment (vishalchangrani) [reply docs/content/node-operation/node-bootstrap.mdx:265]: yup - thanks for the reminder. There is indeed a secure hostname that points to AN7 (`secure.mainnet.nodes.onflow.org`). I will update the doc.

Comment (jordanschalm) [reply docs/content/node-operation/node-bootstrap.mdx:265]: Here is the network key for AN7: `28a0d9edd0de3f15866dfe4aea1560c4504fe313fc6ca3f63a63e4f98d0e295144692a58ebe7f7894349198613f65b2d960abf99ec2625e247b1c78ba5bf2eae`

For reference, here is a script to retrieve the network key from the protocol state snapshot in the right format:
```
cat ./root-protocol-state-snapshot.json | jq '.Identities[] | select(.Address == "access-007.mainnet16.nodes.onflow.org:3569") | .NetworkPubKey' | sed 's/\"//g' | base64 --decode | od -t x1 -An | sed 's/ //g' | tr -d '\n'
```

File: docs/content/node-operation/node-bootstrap.mdx (hunk 3 above)
Comment (jordanschalm) [inline docs/content/node-operation/node-bootstrap.mdx:294]:
```suggestion
A snapshot can be obtained from any Access Node using [Flow CLI](/flow-cli/).
```

Comment (vishalchangrani) [reply docs/content/node-operation/node-bootstrap.mdx:294]: will that work? I didn't see `flow-cli` as a folder under content to make a relative reference. I saw other places in the documentation where flow-cli is referenced using the full url.

Comment (jordanschalm) [reply docs/content/node-operation/node-bootstrap.mdx:294]: I think it should be fine. It works for me locally.

**Comment by jordanschalm** [comment]
@vishalchangrani FYI - pushed one commit (https://github.com/onflow/flow/pull/868/commits/89ec8c27e86d4969af3fe0c4f3975aa0530b59d7) addressing outstanding comments from https://github.com/onflow/flow/pull/862

**Comment by vishalchangrani** [comment]
> @vishalchangrani FYI - pushed one commit ([89ec8c2](https://github.com/onflow/flow/commit/89ec8c27e86d4969af3fe0c4f3975aa0530b59d7)) addressing outstanding comments from #862

lgtm

**Review by jordanschalm — APPROVED** [review]

**Pull request merged**